pkg/services/ngalert/store/alert_rule.go

```diff
@@ -679,3 +692,39 @@ func (st DBstore) validateAlertRule(alertRule ngmodels.AlertRule) error {
 	}
 	return nil
 }
+
+// deleteOldAlertRuleVersions deletes old alert_rule_version DB records for a particular rule in an org
+func (st *DBstore) deleteOldAlertRuleVersions(ctx context.Context, ruleUID string, orgID, limit int64) (int64, error) {
```

Comment:
NIT: Looks like we've split the receivers between pointer and value on DBstore, not sure if it was intentional or not, but it's probably worth staying consistent within the file for now.